### DISCUSSION TITLE
Feross - Socket.dev

### DISCUSSION THREAD

**Discussion by hipstersmoothie** [devtools-fm/devtools.fm#250] [Announcements]
- https://feross.org/
- https://github.com/feross
- https://twitter.com/feross
- https://twitter.com/SocketSecurity
- https://socket.dev/

### Past

- [x] You have a prolific open source career and you're the author of many popular packages. When first got you into it? What drove you to create so many packages? How do you maintain it today?
- [x] You've written a lot on OSS funding. How do you think your past experiments panned out? Do you think the state of the world is better today? How do you think it still needs to improve?
- [x] You were heavily involved with web torrents. How does that work? What were some cool things you built with it? Is wornhole built on it?
- [x] You teach a class at standford? What do you cover?

### Socket

- [x] What is socket? Why did you build it?
- [x] How does it differ from other dependency/security solutions?
- [x] a big problem with products like this is they can be very noisy, how does socket cut through the noise?
- [x] What ecosystems does it cover? Do you all plan to cover more?
- [x] Socket gives a score to a package. What does a score consist of?
- [x] How has AI made making package scores easier? What type of info do the bots give me?
- [x] Is it all positives or are there some false positives that need to be waded through?
- [x] What are some terrifying trends you've seen in package exploits? (ransomware, typo squatting)

### Future

- [x] What's next for socket?
- [x] What's your spiciest dev take?
- [x] Future of web/open source/oss security?

**Comment by just-be-dev** [comment]
Tooltips:
- https://saint11.org/blog/pixel-art-tutorials/
- https://www.cameronmacleod.com/blog/how-does-shazam-work

**Comment by hipstersmoothie** [comment]
NOTES 

https://amzn.to/47QG8WZ
https://github.com/LavaMoat/LavaMoat
